packages/apps/composer-app/vite.config.ts

```diff
@@ -99,7 +99,6 @@ export default defineConfig({
       content: [
         resolve(__dirname, './index.html'),
         resolve(__dirname, './src/**/*.{js,ts,jsx,tsx}'),
-        resolve(__dirname, './node_modules/@dxos/app-framework/src/**/*.{js,ts,jsx,tsx}'),
```

Comment:
i somehow needed to do this -- if you've tested all the dialogs then perhaps this is now resolved.

Reply:
this was because you added ui components to app-framework